.bazelrc

```diff
@@ -31,6 +31,10 @@ build --copt -Wno-deprecated-declarations
 # Envoy's runtime system. This option prevents them from being compiled out.
 build --copt -DABSL_FLAGS_STRIP_NAMES=0
 
+# The defaults are JDK 11 on newer versions of Bazel
```

Comment:
This could potentially live under an Android specific bazelrc config if we'd prefer that not all builds have the JDK pinned to this.

Reply:
In EM this shouldn't matter for non android targets anyways